peps/pep-0791.rst

```diff
@@ -0,0 +1,165 @@
+PEP: 791
+Title: imath --- module for number-theoretic functions
+Author: Sergey B Kirpichev <skirpichev@gmail.com>
+Sponsor: Victor Stinner <vstinner@python.org>
+Discussions-To: Pending
+Status: Draft
+Type: Standards Track
+Created: 10-May-2025
+Python-Version: 3.15
+Post-History: 02-Jun-2019,
```

Comment:
Is there a link for this 2019?

Reply:
Hmm, maybe https://mail.python.org/archives/list/python-ideas@python.org/thread/YYJ5YJBJNCVXQWK5K3WSVNMPUSV56LOR/#YYJ5YJBJNCVXQWK5K3WSVNMPUSV56LOR
is better, this can be linked.

Reply:
I looks that was my link right above.  I'll add that.